src/Categories/CategoryHydrator.php

```diff
@@ -0,0 +1,18 @@
+<?php
+namespace FurnitureStoreApi\Categories;
+use \PDO;
+
+class CategoryHydrator
+{
+    public static function getCategories()
+    {
+        $db = new PDO('mysql:host=db;dbname=furniture_store', 'root', 'password');
```

Comment:
Again, you don't want to create a new database connection every time this method runs! Either create it once outside of here and pass it as an argument to the method, or have a static property for the DB connection that you call a method to set.